src/extension/wmts-multi-dimensional/src/test/java/org/geoserver/gwc/wmts/VectorTimeDimensionTest.java

```diff
@@ -48,6 +49,38 @@ public void testDisabledDimension() throws Exception {
                 DimensionsUtils.extractDimensions(wms, getLayerInfo(), ALL_DOMAINS).size(), is(0));
     }
 
+    @Test
+    public void testFixedValueRangeSingleValueDimension() throws Exception {
+        // enable a time dimension
```

Comment:
What is the purpose of this tests? looks like in the end they are simply testing that the set get fixedValues works...